lib/xeroizer/partner_application.rb

```diff
@@ -42,5 +42,14 @@ def initialize(consumer_key, consumer_secret, path_to_private_key, path_to_ssl_c
         end
       end
     
+    private
+
+      def read_certificate(cert)
+        if cert[-4..-1] == '.pem'
```

Comment:
Why not file.exists? instead. If you have a valid private key but have renamed it, this will fail.